packages/views/src/use-view.ts

```diff
@@ -66,7 +66,7 @@ export function useView( config: ViewConfig ): UseViewReturn {
 	const { set } = useDispatch( preferencesStore );
 
 	const baseView: View = persistedView ?? defaultView;
-	const page = queryParams?.page ?? baseView.page ?? 1;
+	const page = Number( queryParams?.page ?? baseView.page ?? 1 );
```

Comment:
For me the issue should be fixed where we read the value from the url instead of here. But I guess this is defensive coding.